tensorboard/webapp/webapp_data_source/BUILD

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("@npm_angular_bazel//:index.bzl", "ng_module")
+load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
+
+ng_module(
+    name = "webapp_data_source",
+    srcs = [
+        "tb_server.data_source.module.ts",
+        "tb_server.data_source.ts",
+    ],
+    deps = [
+        "//tensorboard/webapp/angular:expect_angular_common_http",
+        "//tensorboard/webapp/types",
+        "@npm//@angular/common",
```

Comment:
I forgot which package provides `@angular/core`. Is this the one? Is `@angular/common' actually being used? We need better linter/checker (gts!) :(

Reply:
`common` is used for `HttpClient`.  I guess it was picking up `core` transitively but I went ahead and included it explicitly for completeness.